### DISCUSSION TITLE
Private chats

### DISCUSSION THREAD

**Discussion by Indious111** [ScratchAddons/ScratchAddons#4540] [Ideas] [answered]
Can we start private chats on Scratch as a Scratch Addon? On the right of the explore box, could we have a button that brings you to private chats, where you can add contacts, but you have to follow the community guidelines or else you will be blocked from private chats for 5 days.

Sincerely, _INDIOUS111
![Indious Flag](https://user-images.githubusercontent.com/100092815/166157915-f2cb2aa2-62d7-4eff-8c18-de288fb4c85d.png)
_

**Comment by CST1229** [comment]
Impossible unless we had an external server, and would proably still be rejected.

**Reply by Norbiros** [reply]
It is possible... It could work like online status, but problem will be with en/decrypting messages... But hard to implement, unnesesary, ST won't like this, and off you want chat just move to something else like discord

**Comment by Weredime** [comment]
What is that picture doing here

**Comment by Endermaster3690** [comment]
ST won't like this

**Reply by Dhuls-Scratch** [reply]
They already don't like us.
Remember that we are a browser extension,

**Reply by Norbiros** [reply]
![Screenshot from 2022-05-03 18-12-42](https://user-images.githubusercontent.com/89837724/166492286-6fbad66e-5b07-4400-9a04-2a68dcd4f8f6.png)

But griffpatch likes us, thats more important.

**Reply by Endermaster3690** [reply]
Dhuls, they just banned mentions of browser extensions.
Sure, we're a browser extension, but that doesn't mean they hate us.

**Reply by Cool-Scratcher** [reply]
> Dhuls, they just banned mentions of browser extensions. Sure, we're a browser extension, but that doesn't mean they hate us.

If I were them I'd be salty that a bunch of teenagers and random people on the internet made a browser extension that is better than vanilla Scratch. They basically hate us, and adding a chat platform will just get their attention.

**Reply by Endermaster3690** [reply]
we don't hate scratch (do we?)
also why would you be salty?
if you don't like sa you can also just not use it

**Reply by Cool-Scratcher** [reply]
> we don't hate scratch (do we?)

No lol I never said that

> also why would you be salty?
> if you don't like sa you can also just not use it

what

**Reply by Endermaster3690** [reply]
> > we don't hate scratch (do we?)
> 
> No lol I never said that

i see what you mean, other way around. Why do you think scratch basically hates us?

> >  also why would you be salty?
> > if you don't like sa you can also just not use it
> 
> what

Why would you be salty that people made browser extensions that improve scratch?

**Comment by apple502j** [comment, marked as answer]
This idea is rejected.
